opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +154,57 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN"
+
+
+class _OpenTelemetryStabilitySignalType:
+    HTTP = "http"
+
+
+class _OpenTelemetryStabilityMode:
+    # http - emit the new, stable HTTP and networking conventions ONLY
```

Comment:
I think it is better to use semantic convention version numbers here, for example, instead of `http`, `HTTP-1.21`, instead of `default` `HTTP-1.20`.